targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_MIMXRT1170/TARGET_EVK/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2021 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+add_library(mbed-evk INTERFACE)
+
+target_include_directories(mbed-evk
+    INTERFACE
+        .
+        xip
+)
+
+target_sources(mbed-evk
+    INTERFACE
+        flash_api.c
+        clock_config.c
+	    mbed_overrides.c
+        PeripheralPins.c
+        pinmap.c
+        serial_api.c
+	    us_ticker.c
```

Comment:
same here, align

Reply:
same

Reply:
Are you using tabs instead of spaces in your editor?